src/DSE.Open.Records.Abstractions/ClinicalConceptCode.cs

```diff
@@ -18,11 +18,12 @@ namespace DSE.Open.Records;
 [EquatableValue]
 [JsonConverter(typeof(JsonSpanSerializableValueConverter<ClinicalConceptCode, long>))]
```

Comment:
Utf8?

And ConditionDiagnosisCode?

and HearingDiagnosisCode?

Reply:
Done.